packages/office-ui-fabric-react/src/components/Modal/Modal.base.tsx

```diff
@@ -4,12 +4,16 @@ import { FocusTrapZone, IFocusTrapZone } from '../FocusTrapZone/index';
 import { animationDuration, getOverlayStyles } from './Modal.styles';
 import { IModalProps, IModalStyleProps, IModalStyles, IModal } from './Modal.types';
 import { Overlay } from '../../Overlay';
-import { Layer } from '../../Layer';
+import { ILayerProps, Layer } from '../../Layer';
 import { Popup } from '../Popup/index';
 import { withResponsiveMode, ResponsiveMode } from '../../utilities/decorators/withResponsiveMode';
 
 // @TODO - need to change this to a panel whenever the breakpoint is under medium (verify the spec)
 
+const DefaultLayerProps: ILayerProps = {
+  eventBubblingEnabled: false
+};
```

Comment:
Is this a holdover from a previous approach? I don't see this used anywhere...

Reply:
`DefaultLayerProps` on line 129 of `Modal.base.tsx` where it its spread to `mergedLayerProps`.

Reply:
I misread this as `defaultProps`